helper/staking/staking.go

```diff
@@ -2,18 +2,14 @@ package staking
 
 import (
 	"fmt"
-	"github.com/0xPolygon/polygon-edge/chain"
-	"github.com/0xPolygon/polygon-edge/helper/hex"
 	"math/big"
 
+	"github.com/0xPolygon/polygon-edge/chain"
+	"github.com/0xPolygon/polygon-edge/helper/hex"
 	"github.com/0xPolygon/polygon-edge/helper/keccak"
 	"github.com/0xPolygon/polygon-edge/types"
 )
 
-var (
-	StakingSCAddress = types.StringToAddress("1001")
```

Comment:
How do we know the staking SC address if it's not hardcoded?

Reply:
I removed because this address is defined in multiple places. You can find same variable in contracts/staking package